src/CommunityToolkit/Xamarin.CommunityToolkit/Effects/Touch/PlatformTouchEffect.uwp.cs

```diff
@@ -160,49 +160,49 @@ void OnPointerCaptureLost(object? sender, PointerRoutedEventArgs e)
 			isPressed = false;
 
 			if (effect.Status != TouchStatus.Canceled)
-				effect.HandleTouch(TouchStatus.Canceled);
+				effect?.HandleTouch(TouchStatus.Canceled);
 
-			effect.HandleUserInteraction(TouchInteractionStatus.Completed);
+			effect?.HandleUserInteraction(TouchInteractionStatus.Completed);
 
 			if (effect.HoverStatus != HoverStatus.Exited)
```

Comment:
we need `effect?.` here as well - this would fix the `CS8602: Dereference of a possibly null reference` issue in the build pipeline

Reply:
Thanks)